packages/bygger/src/Forms/index.test.jsx

```diff
@@ -1,59 +1,35 @@
-import NewFormPage from "./NewFormPage";
-import NavFormBuilder, { UnstyledNavFormBuilder } from "../components/NavFormBuilder";
-import waitForExpect from "wait-for-expect";
-import { FakeBackendTestContext } from "../testTools/frontend/FakeBackendTestContext";
```

Comment:
Sjekk om FakeBackendTestContext har gått ut på dato

Reply:
Slettet denne og relatert kode for testoppsett